### PR TITLE
move load_pickle to xshards

### PR DESCRIPTION
**Pull request by jenniew**
Move load_pickle from SparkXShards to XShards.

Related issue: https://github.com/analytics-zoo/orca/issues/62

### CODE REVIEW

File: pyzoo/zoo/orca/data/shard.py [hunk 1]
```diff
@@ -267,10 +278,7 @@ def save_pickle(self, path, batchSize=10):
         self.rdd.saveAsPickleFile(path, batchSize)
         return self
 
-    @classmethod
-    def load_pickle(cls, path, minPartitions=None):
-        sc = init_nncontext()
-        return SparkXShards(sc.pickleFile(path, minPartitions))
+
```
Comment (hkvision) [inline pyzoo/zoo/orca/data/shard.py:281]: remove this empty line

File: pyzoo/zoo/orca/data/shard.py [hunk 2]
```diff
@@ -47,6 +47,17 @@ def num_partitions(self):
         """
         pass
 
+    @classmethod
+    def load_pickle(cls, path, minPartitions=None):
+        """
+        Load XShards from pickle files
```
Comment (hkvision) [inline pyzoo/zoo/orca/data/shard.py:53]: missing `.` at the end

File: pyzoo/zoo/orca/data/shard.py [hunk 3]
```diff
@@ -47,6 +47,17 @@ def num_partitions(self):
...
+        :param path: The pickle file path/directory
+        :param minPartitions: minimum partitions for the XShards
```
Comment (hkvision) [inline pyzoo/zoo/orca/data/shard.py:55]: The minimum number of partitions ...

**Comment by hkvision** [comment]
LGTM

File: pyzoo/zoo/orca/data/shard.py (hunk 1 above)
Comment (jenniew) [reply pyzoo/zoo/orca/data/shard.py]: updated.

**Comment by jenniew** [comment]
Jenkins pass: http://10.239.47.210:18888/view/ZOO-PR/job/ZOO-PR-Validation/3866/

**Pull request merged**